src/lib/configs/project/service.ts

```diff
@@ -205,6 +206,10 @@ const getDefault: (
   relativePathToFacade: string,
   name: string,
 ): GetDefaultConfig => {
+  // All paths in config should be kept in platform independent way for consistency
+  // Also yaml parser would complain about backslashes, so it's better to avoid them instead of shielding
```

Comment:
Good idea to keep these paths posix everywhere. Keep in mind that there are multiple places where the same problem happens (cli accepts paths in different places)
probably good idea to have a simple function that converts a path to posix and use it everywhere